src/overlays/actors/ovl_En_Mag/z_en_mag.h

```diff
@@ -8,9 +8,51 @@ struct EnMag;
 
 typedef struct EnMag {
     /* 0x0000 */ Actor actor;
-    /* 0x014C */ char unk_14C[0xE1DC];
+    /* 0x014C */ char unk_14C[0x0004];
+    /* 0x0150 */ u8 unk_150[0xE188]; // kanfont struct/buffer
+    /* 0xE2D8 */ s16 effectFadeInState;
+    /* 0xE2DA */ s16 effectFadeInTimer;
+    /* 0xE2DC */ s16 globalState;
+    /* 0xE2E0 */ f32 effectPrimLodFrac;
+    /* 0xE2E4 */ f32 effectPrimColor[3];
+    /* 0xE2F0 */ f32 effectEnvColor[3];
+    /* 0xE2FC */ f32 effectAlpha;
+    /* 0xE300 */ f32 mainAlpha;
+    /* 0xE304 */ f32 subAlpha;
```

Comment:
I named them that way because "main" is used for the main title (the big logo and "The Legend of"/"Ocarina of Time" parts), while "sub" is used for the "Master Quest" sub title. That's about it I think.